cde-core/resource/js/cdf-dd.js

```diff
@@ -1881,11 +1881,12 @@ $(function() {
     },
     content: function(data, settings, original) {
       // If it is string assume it is an array.
+      var json;
       if(String == data.constructor) {
-        eval('var json = ' + data);
+        json = JSON.parse(data);
```

Comment:
@akhayrutdinov I don't feel comfortable with that solution, I'd say usages of jeditable must be tracked down, and It must be insured that the json String provided is sane (at least where we have control over it).
For this particular error, I'd do this -> https://github.com/jvelasques/cde/commit/6042069de21397e506305594abd42330864b10d7